index.html

```diff
@@ -12107,17 +12107,17 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
-				<p>The <code>aria-brailleroledescription</code> property gives authors the ability to override how <a>assistive technologies</a> localize and express the name of a role in Braille. Thus inappropriately using <code>aria-brailleroledescription</code> may inhibit users' ability to understand or interact with an element on Braille interfaces. Authors SHOULD limit use of <code>aria-brailleroledescription</code> to clarifying the purpose of non-interactive container roles like <rref>group</rref> or <rref>region</rref>, or to providing a <em>more specific</em> description of a <rref>widget</rref> in a Braille context.</p>
-				<p>Authors MUST NOT use <code>aria-brailleroledescription</code> without providing a <em>more specific</em> <code>aria-roledescription</code>. In general, <code>aria-brailleroledescription</code> should only be used in rare cases when a <code>aria-roledescription</code> does not suffice.</p>
+				<p>The <code>aria-brailleroledescription</code> property gives authors the ability to override how <a>assistive technologies</a> localize and express the name of a role in Braille. Thus inappropriately using <code>aria-brailleroledescription</code> may inhibit users' ability to understand or interact with an element on braille interfaces. Authors SHOULD limit use of <code>aria-brailleroledescription</code> to clarifying the purpose of non-interactive container roles like <rref>group</rref> or <rref>region</rref>, or to providing a <em>more specific</em> description of a <rref>widget</rref> in a braille context.</p>
+				<p>Authors MUST NOT use <code>aria-brailleroledescription</code> without providing a <em>more specific</em> <code>aria-roledescription</code>. In general, <code>aria-brailleroledescription</code> should only be used in rare cases when a <code>aria-roledescription</code> is excessively verbose when rendered in Braille.</p>
 				<p> When using <code>aria-brailleroledescription</code>, authors SHOULD also ensure that:</p>
 				<ol>
 					<li>The element to which <code>aria-brailleroledescription</code> is applied has a valid <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> role or has an implicit WAI-ARIA role semantic.</li>
 					<li>The value of <code>aria-brailleroledescription</code> is not empty or does not contain only whitespace characters.</li>
 					<li>The value of <code>aria-brailleroledescription</code> consists of a string of either Unicode with no Unicode Braille Patterns (U+2800..U+28FF) or consists of a string of only Unicode Braille Patterns (U+2800..U+28FF) while not only containing Braille Pattern dots-0 (U+2800).</li>
```

Comment:
The word "either" is logically misplaced.

I don't understand the last phrase:
>while not only containing Braille Pattern dots-0 (U+2800)

I assume that is the code for an empty cell and this means that the string cannot be only empty cells.

If this is correct, I suggest the following:

```suggestion
					<li>The value of <code>aria-brailleroledescription</code> either consists of a string of Unicode that does not include any Unicode Braille Patterns (U+2800..U+28FF) or consists of a string including only Unicode Braille Patterns (U+2800..U+28FF) that would not be an empty string if all Braille Pattern dots-0 (U+2800) were removed.</li>
```